.github/workflows/main.yml

```diff
@@ -189,9 +208,22 @@ jobs:
           cd /tmp/xcb-errors
           sudo make install
 
+      # Only LGI git master supports Lua 5.4. We need to build it from source.
+      - name: Build and Install LGI
+        if: matrix.lua_version == '5.4'
+        run: |
+          wget -O /tmp/lgi.zip https://github.com/pavouk/lgi/archive/refs/heads/master.zip
+          unzip /tmp/lgi.zip -d /tmp
+          cd /tmp/lgi-master
+          sed -i 's/5.1/5.4/' lgi/Makefile
```

Comment:
I'm not very happy with this. But without this change, the installation target is lua5.1 directories. I guess I have to understand how LGI build works to do something better...

Reply:
>  guess I have to understand how LGI build works to do something better...

I doubt there is something very much better. Random data points:

- https://github.com/archlinux/svntogit-community/blob/fd394d24539eab3b51ddb7baa6c36cdbc4a149e7/trunk/PKGBUILD#L33-L35
- I tried to figure out what debian does, but failed to actually find it.

Reply:
I usually do

    make PREFIX=/usr LUA_VERSION=5.4 && make install LUA_VERSION=5.4 PREFIX=/usr

That's the proper way of making it work. Obviously the Makefile could/should parse the result of `pkg-config` and figure this out on its own, but this is not a huge issue.